2022 - LA - Night Lords.cat

```diff
@@ -3736,6 +3747,18 @@
                     <characteristic name="Ld" typeId="e8a6-1da9-d384-8727">8</characteristic>
                     <characteristic name="Save" typeId="e593-6b3c-f169-04f0">3+</characteristic>
                   </characteristics>
+                  <modifierGroups>
+                    <modifierGroup type="and">
+                      <modifiers>
+                        <modifier type="set" value="2" field="57ee-1126-32a9-5672" join=""/>
+                        <modifier type="set" value="Pano 5.3 pg 88" field="annotation"/>
+                        <modifier type="set" value="22" field="d2ee-04cb-5f8a-2642" join=""/>
```

Comment:
The points shouldn't be set here, setting points on a profile doesn't do anything. Also the base points adjustment for this unit should be handled differently. The points per model going up 4ppm means you should add 16 less points to the base points cost. 

This unit is also set up weird in the base data, with the headsman having 18pts instead of the entire unit tax, and then the rest of the unit tax being on the unit. But that can be worked around.